frontend/javascripts/oxalis/view/context_menu.tsx

```diff
@@ -1094,8 +1100,7 @@ function InfoMenuItem(propsWithEventKey: MenuItemProps) {
    * warning.
    */
 
-  const { children } = propsWithEventKey;
-  return <div className="node-context-menu-item">{children}</div>;
+  return { key, label, type: "group" };
```

Comment:
the above comment is not valid anymore. you could mention instead that the trick is simply to use the "group" type to get the desired behavior.